xv83.md

```diff
@@ -35,16 +27,47 @@ xv83/
 │   ├── kd7073/
 │   ├── mxt599/
 │   └── tm4888/
+
+xv84/
+├── admin/
+├── shared/
```

Comment:
@chloemackallah Is `shared` redundant here because all of `xv84` will be shared by definition?

Reply:
Yes very true! Let's get rid of 'shared' and bring everything in it up a level